tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1918,6 +1918,14 @@ INSTANTIATE_TEST_CASE_P(
                        ::testing::Values(DT_FLOAT, DT_HALF, DT_INT32),
                        ::testing::Values(TrtPrecisionMode::FP32)));
 
+// Base class for tests that need to be tested in TrtPrecisionMode::INT8
+class OpConverterTest_Int8Mode : public ParameterizedOpConverterTestBase {};
```

Comment:
I don't like this design this will become messy quickly.

Could we design a better way to handle input and TRT Precision Mode ?

It doesn't make any sense IMHO to come up with:

- `OpConverter_FP32_Test_FP32Mode`
- `OpConverter_FP32_FP16_Test_FP32Mode`
- `OpConverter_FP32_FP16_INT32_Test_FP32Mode`
and
- `OpConverter_FP32_Test_FP16Mode`
- `OpConverter_FP32_FP16_Test_FP16Mode`
- `OpConverter_FP32_FP16_INT32_Test_FP16Mode`
and
- `OpConverter_FP32_Test_INT8Mode`
- `OpConverter_FP32_FP16_Test_INT8Mode`
- `OpConverter_FP32_FP16_INT32_Test_INT8Mode`

There gotta be a better way than this